tap_zendesk/streams.py

```diff
@@ -59,10 +60,16 @@ class Stream():
     replication_key = None
     key_properties = KEY_PROPERTIES
     stream = None
+    request_timeout = None
 
     def __init__(self, client=None, config=None):
         self.client = client
         self.config = config
+        # Set and pass request timeout to config param `request_timeout` value.
+        # If value is 0,"0","" or not passed then it set default to 300 seconds.
+        config_request_timeout = self.config.get('request_timeout')
+        self.request_timeout = (config_request_timeout and float(config_request_timeout)) or REQUEST_TIMEOUT # pylint: disable=consider-using-ternary
```

Comment:
@prijendev We should figure out an alternative to resolve the pylint error. Why to disable it completely?
Can you research on what can be the alternative ways to assign values to `self.request_timeout`?

Reply:
Removed disable of pylint error. Updated code.